ql/src/java/org/apache/hadoop/hive/ql/io/orc/VectorizedOrcAcidRowBatchReader.java

```diff
@@ -2039,4 +2189,29 @@ private static IntegerColumnStatistics deserializeIntColumnStatistics(List<OrcPr
         deserializeIntColumnStatistics(colStats, OrcRecordUpdater.ROW_ID + 1);
     return getKeyInterval(columnStatsArray);
   }
+
+  private static class RowIsDeletedColumnVector extends LongColumnVector {
+    void clear() {
+      Arrays.fill(vector, 0);
```

Comment:
Believe reset() method is a better choise, especially as we set repeat flag below

Reply:
Removed this method since `fill(false)` do what we need